product_docs/docs/pem/8/registering_agent.mdx

```diff
@@ -58,6 +58,58 @@ Before any changes are made on the PEM database, the connecting agent is authent
 
 If you don't provide the password, a password authentication error occurs. You are prompted for any other missing required information. When the registration is complete, the server confirms that the agent was successfully registered.
 
+Each registered PEM agent must have a unique agent ID. For 8.7 and later, 
+
+-   The value `max(id)+1` is assigned to each agent ID. 
+-   You can override the agent ID manually using the command line.
+
+!!! Note
+    For 8.6 and earlier, the sequence `pem.agent_id_seq` generates and gives a unique ID to the PEM agent. 
+
+### Examples
+
+This example shows how to register the PEM agent overriding the default configurations using the `-o` option for 8.7 and later.
+
+Register the PEM agent using command line. Assign agent ID 8 to override the default.
```

Comment:
For your second sentence, it would help to be a little more closely aligned with the code, like maybe:
Assign an `agent_id` value of `8` using the `-o` option.